notebooks/json_blocks/LUT/BLOCK-T401.ipynb

```diff
@@ -0,0 +1,200 @@
+{
```

Comment:
<p>Do you not need <code>name</code> here too as you have in the v2 notebook? </p>
<br/>

 _Reply via <a href='https://app.reviewnb.com/lsst-sitcom/ts_aos_analysis/pull/28/discussion/'>ReviewNB</a>_ <div id='ReviewNBCommentContext-DoNotDelete' style='display:none' data-state='OPEN' data-cellIndex='2'/>

Reply:
No, because for this one the name coincides with the program